### DISCUSSION TITLE
Website UX Improvement

### DISCUSSION THREAD

**Discussion by ArtifactyNight** [cosscom/coss#176] [Suggestions]
**1. View Code Directly from the Website**
Provide the ability to inspect and view the code directly from the website without needing to manually copy it from the source code. This feature would make it easier to select specific parts of the code for reuse or reference without additional steps

**2. Display Component IDs**
Add an option to display unique component IDs (e.g., 1switch-1`) visibly in the corner of each component.  This feature would make it allow to track between website and component on respository

Thank you for considering my suggestions! I appreciate your efforts in improving the project and making it more user-friendly. 🙏

**Comment by guibzo** [comment]
Nice!
I would also add in the "Copy" option so it only copies the actual component, instead of label (showed only in the previews) and such containers, and remove the id (which is most likely unnecessary to be copied).
